src/pages/MarkdownPage.tsx

```diff
@@ -29,7 +30,7 @@ const MarkdownPage: React.FC<MarkdownProps> = ({ loginCallback }) => (
         <CardBody>
           <p>
             CubeCobra supports regular Markdown as well as some extra features specific to our site. If you need any
-            help regarding how to use markdown, please <a href="/contact">contact us</a>.
+            help regarding how to use markdown, please <Link href="/contact">contact us</Link>.
```

Comment:
I noticed links on the markdown explanation page weren't styled as links anymore, so converted to Link component.
Before:
![Capture](https://github.com/user-attachments/assets/62103498-c849-4dee-8d4a-bb33804fe22b)

After:
![Capture2](https://github.com/user-attachments/assets/701e65e2-aca4-4ca8-b5e2-5abd86f373a6)